pkg/alluxiocluster/update_status.go

```diff
@@ -68,9 +69,12 @@ func UpdateStatus(alluxioClusterCtx AlluxioClusterReconcileReqCtx) (ctrl.Result,
 	return ctrl.Result{RequeueAfter: 2 * time.Minute}, nil
 }
 
-func ClusterReady(ctx AlluxioClusterReconcileReqCtx) bool {
+func ClusterReady(ctx *AlluxioClusterReconcileReqCtx) bool {
```

Comment:
```suggestion
func IsClusterReady(ctx *AlluxioClusterReconcileReqCtx) (bool, error) {
```
i think there are errors to propagate

Reply:
Here we don't care about the error - even if there is error the following code needs to be executed. But it is true that maybe in the future some code needs the returned error.